.drone.yml

```diff
@@ -2362,52 +2443,73 @@ trigger:
 workspace:
   path: /dev/shm/tmp
 
+# use ramfs for go build cache
+# note: declaring the WORKSPACE_DIR variable once in this block and then trying to
+# quote it in commands below doesn't seem to work; it has to be redeclared as part of
+# the environment inside every individual step. per-pipeline environments seem to be
+# inconsistently implemented across different drone runners and may well be deprecated
```

Comment:
From the docs:
```
Note this feature is only available to Docker pipelines at this time.
```
https://github.com/drone/docs/blob/9488eb2c218f9903bd8991e3b4fdf240236c2c5b/content/pipeline/environment/syntax.md
I'd assume due to that.

Reply:
Yes, except this per-pipeline functionality _does_ actually work in Kubernetes and we make use of it throughout our pipeline (purely because it's less lines than respecifying the Go runtime version as an environment variable inside every step that needs it)

I presume what they're actually saying is that they don't make any guarantees it will work with anything other than Docker, which is fine.